src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/goal/Goal.kt

```diff
@@ -22,10 +22,10 @@ class Goal(
   val category: GoalCategory,
   var status: GoalStatus = GoalStatus.ACTIVE,
   val notes: String? = null,
-  val createdBy: String,
-  val createdAt: Instant,
-  val lastUpdatedBy: String,
-  val lastUpdatedAt: Instant,
+  val createdBy: String?,
+  val createdAt: Instant?,
+  val lastUpdatedBy: String?,
+  val lastUpdatedAt: Instant?,
```

Comment:
Controversial, but unless we introduce a DTO here then I'm not sure what else to do 🤔